bfe.json

```diff
@@ -4,45 +4,45 @@
     "type": "feed",
     "sigil": "@",
     "formats": [
-      { "code": 0, "format": "ssb/classic",     "data_length": 32, "suffix": ".ed25519"   },
-      { "code": 1, "format": "ssb/gabby-grove", "data_length": 32, "suffix": ".ggfeed-v1" },
-      { "code": 2, "format": "bamboo",          "data_length": 32, "suffix": ".bamboo" },
-      { "code": 3, "format": "ssb/bendy-butt",  "data_length": 32, "suffix": ".bbfeed-v1" },
-      { "code": 4, "format": "ssb/fusion-identity", "data_length": 32, "suffix": ".fusion-v1" }
+      { "code": 0, "format": "classic",     "data_length": 32, "suffix": ".ed25519"   },
+      { "code": 1, "format": "gabby-grove", "data_length": 32, "suffix": ".ggfeed-v1" },
+      { "code": 2, "format": "bamboo",      "data_length": 32, "suffix": ".bamboo" },
+      { "code": 3, "format": "bendy-butt",  "data_length": 32, "suffix": ".bbfeed-v1" },
+      { "code": 4, "format": "fusion-identity", "data_length": 32, "suffix": ".fusion-v1" }
     ]
   },
   {
     "code": 1,
     "type": "msg",
     "sigil": "%",
     "formats": [
-      { "code": 0, "format": "ssb/classic",     "data_length": 32, "suffix": ".sha256"   },
-      { "code": 1, "format": "ssb/gabby-grove", "data_length": 32, "suffix": ".ggmsg-v1" },
-      { "code": 2, "format": "cloaked msg",     "data_length": 32, "suffix": ".cloaked" },
-      { "code": 3, "format": "bamboo",          "data_length": 64, "suffix": ".bamboo" },
-      { "code": 4, "format": "ssb/bendy-butt",  "data_length": 32, "suffix": ".bbmsg-v1" }
+      { "code": 0, "format": "classic",     "data_length": 32, "suffix": ".sha256"   },
+      { "code": 1, "format": "gabby-grove", "data_length": 32, "suffix": ".ggmsg-v1" },
+      { "code": 2, "format": "cloaked",     "data_length": 32, "suffix": ".cloaked" },
+      { "code": 3, "format": "bamboo",      "data_length": 64, "suffix": ".bamboo" },
+      { "code": 4, "format": "bendy-butt",  "data_length": 32, "suffix": ".bbmsg-v1" }
     ]
   },
   {
     "code": 2,
     "type": "blob",
     "sigil": "&",
     "formats": [
-      { "code": 0, "format": "ssb/classic", "data_length": 32, "suffix": ".sha256" }
+      { "code": 0, "format": "classic", "data_length": 32, "suffix": ".sha256" }
     ]
   },
   {
     "code": 3,
-    "type": "diffie-hellman",
+    "type": "key",
```

Comment:
Maybe "encryption key" (was that what we agreed on in the meeting? I don't accurately remember)? "key" is a very generic word.

Reply:
This PR then automatically means you're proposing the SSB URI `ssb:keys/box2-dm-dh/` and I believe `ssb:keys/` is going to confuse a lot of people on what it means (given the existence of `ssb-keys`). Why not `ssb:encryption-keys/box2-dm-dh` ?

Reply:
Mainly because it's hella long, and I wasn't to save my bytes!

I'm up for something different to reduce confusion. Couldn't think of anything right though. Maybe emoji? Joking

Reply:
It's not prohibitively long. Remember SSB URIs are for human readability (while still computer parse-able), and we have other long SSB URIs such as `ssb:experimental?action=claim-http-invite&.....`